packages/account-v2/src/components/SignupWizard/index.scss

```diff
@@ -0,0 +1,27 @@
+.ReactModal__Overlay {
+    background-color: rgba(0, 0, 0, 0.72);
+    z-index: 9999;
+}
+
+.ReactModal__Content {
+    opacity: 0;
```

Comment:
cant we set these in tailwind?

Reply:
these are specific to the react modal so these should be specified here as mentioned in the documentation

Reply:
Can u try using css modules?